app.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+const path = require('path');
+
+const bodyParser = require('body-parser');
+const express = require('express');
+
+const config = require('./app/config/').Config;
+
+const ErrorHandler = require('./app/util/').ErrorHandler;
+
+const database = require('./app/database/').Database;
+
+const api = require('./app/api/api');
+
+const app = express();
+
+
+/**
+ * @description Serve static files.
+ */
+app.use(express.static(path.join(__dirname, 'public')));
```

Comment:
No need for this, as we only want the API and not static files :) Maybe I was a bit unclear about this in the original spec :)